### PR TITLE
Unified Configurations and tweaked some parameters, improved README

### PR DESCRIPTION
**Pull request by brandstaetter**
### Description

Unified configs, improved README

### CODE REVIEW

File: Marlin/src/config/examples/delta/Anycubic/Kossel/README.md [hunk 1]
```diff
@@ -1,40 +1 @@
...
It is very important to follow the correct procedure to set it up after flashing the firmware, 
otherwise you might damage the printer by ramming the nozzle into the buildplate:

* Configuration >Advanced Settings > Initialize EEPROM
* Motion > Move Axis > Soft Endstops : Off
* Auto Home and slowly move the nozzle down until it barely touches the bed (do the paper test: a normal sheet of paper should just feel the drag of the nozzle) and note this number
* subtract this number from the value in Configuration > Delta Calibration > Delta Settings > Height (if it's negative, add it)
* save and try the paper test again to verify your height
* Configuration > Store Settings
* Motion > Bed Leveling (using paper test)
```
Comment (thinkyhead) [inline Marlin/src/config/examples/delta/Anycubic/Kossel/README.md]: What's up with these line-endings? We use UNIX line-endings.

Comment (brandstaetter) [reply Marlin/src/config/examples/delta/Anycubic/Kossel/README.md]: Weird, my editor might do some weird crap. Fixing now.

**Comment by thinkyhead** [comment]
It's kind of bad to overdo the unification of configs. It makes it harder to keep configurations in sync as new features are added. If there are more than a few settings that should apply to one machine versus the other, then it's better to split them up. Especially as users may not like the options that are pre-configured, and shouldn't have to wade through extra conditionals to apply their own preferences.

**Comment by brandstaetter** [comment]
Thanks for the feedback!

my reasoning was: there are these 3 things that make the main difference between the Anycubic Kossels out there:

plus: bigger, affects 3 values in the delta kinematics
probe: autolevel and autocalibration available, also 2 different probes that just affect the z-offset
heated bed: could be removed if it makes it too complicated, but just activates the heatbed thermistor and sets 2 different BEDPID value sets, because there are 2 different beds in the wild.

if you think it's too unified, I can de-simplify it again. but splitting normal and plus does not make sense IMHO, as there are only the dimensions different.

**Comment by thinkyhead** [comment]
It's not too bad. I'll clean it up slightly and see how it looks.

**Comment by thinkyhead** [comment]
You might want to double-check the `README.md` and make sure it's accurate now. It still seems to mention multiple configs.

**Pull request merged**